cmd/goal/clerk.go

```diff
@@ -389,6 +397,44 @@ var sendCmd = &cobra.Command{
 			}
 		}
 
+		// Handle the case where the user wants to send to an account that was rekeyed to a multisig account
+		if msigParams != "" {
+			// Decode params
+			params := strings.Split(msigParams, " ")
```

Comment:
do you have to use quotation marks when using this flag? if so, maybe the split character should be a comma instead

Reply:
yes, unfortunately, there is no way around it. The spaces are there to keep it consistent with the other commands (rather than commas)